extensions/wikia/DesignSystem/services/templates/DesignSystemGlobalNavigationOnSiteNotifications_index.php

```diff
@@ -1,19 +1,25 @@
-<div id="onSiteNotificationsEntryPoint"
-     class="wds-global-navigation__notifications-menu wds-global-navigation__notifications-menu wds-dropdown notifications-container">
-	<div class="wds-dropdown__toggle wds-global-navigation__dropdown-toggle"
+<div class="wds-global-navigation__notifications-menu wds-dropdown notifications-container">
+	<div id="on-site-notifications-dropdown"
+	     class="wds-dropdown__toggle wds-global-navigation__dropdown-toggle"
 	     title="<?= wfMessage( 'global-navigation-notifications-title' )->escaped() ?>">
 		<div class="bubbles">
```

Comment:
Do you plan to get rid of this class and `notifications-container` before finishing the work? I don't think we should stick to the legacy code if not necessary.